src/components/Quiz.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Quiz extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: []
+    };
+  }
+
+  getQuizzes() {
+    return axios.get('/quizzes', {
+    })
+      .then((response) => {
+        this.setState({
+          quizzes: response.data.quizzes,
+          questions: response.data.quizzes[0].questions
+        });
+        console.log(response);
+      }).catch(function() {
+        console.log('get request failed');
+      });
+  }
+
+  componentDidMount() {
+    this.getQuizzes();
+  }
+
+  renderQuizTitle(state) {
+    return state.map(quiz =>
+      <section className="quiz-title" key={quiz.id}>
+        <h1>{quiz.title}</h1>
+      </section>
+    )
+  }
+
+  renderQuestions(state) {
+    return state.map(questions =>
+      <section className="questions" key={questions.id}>
+        <h2 className="question-title">
+          {questions.title}
+        </h2>
+        <article className="answers-container">
+          {questions.answers.map((answers, i) =>
```

Comment:
We might want to break up this block of HTML we're returning by creating some additional components. If we add a Question component and an Answer component, we could render something like the following:

```javascript
questions.map((question, index) =>
    <Question title={question.title} key={question.id}/>
    (question.answers).map(answer =>
        <Answer title={answer.title} />
    )
(That is not going to be the exact syntax, and I'm sure it will break in all sorts of ways, but that's what we should be going for here.)